app/assets/javascripts/app/controllers/ticket_zoom.coffee

```diff
@@ -1221,6 +1225,26 @@ class App.TicketZoom extends App.Controller
       overview_id: @overview_id
     )
 
+  copyTicketNumber: =>
+    text = $('.active.content .js-objectNumber').first().data('number') || ''
```

Comment:
IMO this should be `text = @el.find('.js-objectNumber').first().data('number') || '' ` to only query the local @el and not whole browser dom.